src/dune

```diff
@@ -0,0 +1,14 @@
+(melange.emit
+ (target output)
+ (alias react)
+ (libraries melange reason_react world)
+ (entries :standard \ hello)
+ (compile_flags :standard -bs-jsx 3)
+ (module_systems es6))
+
+(melange.emit
+ (target node)
+ (alias node)
+ (libraries world)
+ (entries hello)
```

Comment:
should we upgrade dune once more and get `(modules .. )` for this? was merged this morning